indico/web/client/js/custom_elements/ind_date_picker.scss

```diff
@@ -53,6 +53,12 @@ ind-date-picker {
   }
 }
 
+.ui-widget ind-date-picker > button {
```

Comment:
@foxbunny Do you see a better solution for this? Making the rule inside `%text-size-reset` important would also work, but not sure if this doesn't have some bad side effects elsewhere...

Reply:
%text-size-reset is probably not the best workaround for the font size issue. It's more like a marker so I know what to address once we actually solve the problem (god knows when, lol). In theory, it shouldn't have any bad side-effects, but it can fail to work when, say, there are rules with high-specificity selectors that affect elements inside the widget and override font sizes.